proposals/initial-connection/README.md

```diff
@@ -255,6 +255,13 @@ Fields that override the starting default for all the elements of that kind.
 
 These set a default for the whole Thing
 
+| Vocabulary Term | Description | Assignment | Type | Remarks |
+|---|---|---|---|---|
+| connection | A reference to or an in-place definition of a connection definition, if missing a protocol binding default is in place. | optional | String pointing to the `connectionDefinitions` map or Object with type `Connection`. | If string, it MUST refer to a first-level key in `connectionDefinitions`. |
```

Comment:
```suggestion
| connection | A reference to or an in-place definition of a connection definition, protocol bindings MAY provide default definitions. | optional | String pointing to the `connectionDefinitions` map or Object with type `Connection`. | If string, it MUST refer to a first-level key in `connectionDefinitions`. |
```

Maybe?

Reply:
That is a discussion to be had :D I want to make this PR rather straightforward with some basic text wrangling

Reply:
it is in any case an improvement I'd say.

Reply:
I think that defaults will be possible in all cases but I want to have it as a separate discussion. All cases meaning HTTP binding can say the default http version, tls version. This affects the connection term. security can be also affected in a way but these are then some longer discussions. Until we know how the defaults are going to be specified, no need to add an assertion

Reply:
Sorry I have misunderstood something here... I am not sure what we meant here in the beginning. I will copy over but create an issue to follow up.